xblocks_contrib/poll/poll.py

```diff
@@ -0,0 +1,78 @@
+"""TO-DO: Write a description of what this XBlock is."""
+
+from importlib.resources import files
+
+from web_fragments.fragment import Fragment
+from xblock.core import XBlock
+from xblock.fields import Integer, Scope
+
+
+class PollXBlock(XBlock):
+    """
+    TO-DO: document what your XBlock does.
+    """
```

Comment:
Can you add a comment explaining that this block just exists to demonstrate the structure of the repository and that it is still a work-in-progress?

Reply:
I've added comments to each XBlock explaining that they're meant to demonstrate the structure.